iroha/src/account.rs

```diff
@@ -147,21 +147,25 @@ pub mod isi {
 
     impl Transfer<Account, Asset, Account> {
         fn execute(&self, world_state_view: &mut WorldStateView) -> Result<(), String> {
-            world_state_view
+            let from = world_state_view
```

Comment:
`from` sounds a lot like a std function, maybe name it `from_account`?

Reply:
maybe I will rename it to `source`